### DISCUSSION TITLE
Improvements for the Social Sharing block.

### DISCUSSION THREAD

**Discussion by cristian-ungureanu** [Codeinwp/otter-blocks#1484] [Ideas & Feedback]
One of our users tested Neve FSE, but he's missing some features from Neve's post sharing buttons. When I recommended the Social Sharing block, our user said this: 

`Thanks. WhatsApp, email is missing, and it would be great if I can add a custom icon with '#comments' for people to scroll to the comments section.`

Link: https://www.facebook.com/groups/nevecommunity/posts/1725525801182652?comment_id=1725822287819670&reply_comment_id=1727753874293178&notif_id=1675337037522340&notif_t=group_comment&ref=notif

**Comment by HardeepAsrani** [comment]
@cristian-ungureanu I think, overall, all three of these make sense, we can have all three of them. I'll move it to issues.
